### PR TITLE
applications: nrf_desktop: Slave latency for dongle connection

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds slave latency for dongle connection.
Log informing about updated latency was moved to ble_state (to make it visible also on dongle).

### CODE REVIEW

**Comment by MarekPieta** [comment]
@ppryga, could you review?

File: applications/nrf_desktop/src/modules/hid_forward.c [hunk 1]
```diff
@@ -318,16 +404,177 @@ static u8_t hidc_read_cfg(struct bt_gatt_hids_c *hidc,
...
 	}
 	return NULL;
 }
 
+static int switch_to_low_latency(struct hids_subscriber *subscriber)
+{
+	int err;
+
+	if (IS_ENABLED(CONFIG_BT_LL_NRFXLIB)) {
+		if (subscriber->timestamp == 0){
```
Comment (pdunaj) [inline applications/nrf_desktop/src/modules/hid_forward.c]: style `) {`

File: applications/nrf_desktop/src/modules/hid_forward.c [hunk 2]
```diff
@@ -318,16 +404,177 @@ static u8_t hidc_read_cfg(struct bt_gatt_hids_c *hidc,
...
+
+	if (err) {
+		notify_config_forwarded(CONFIG_STATUS_WRITE_ERROR);
+		return;
+	}
+
+	if (forward_pending) {
+		LOG_DBG("GATT read already pending");
+		return;
+	} else {
```
Comment (pdunaj) [inline applications/nrf_desktop/src/modules/hid_forward.c]: no need for else (and increased indent) if we return in line 541

File: applications/nrf_desktop/src/modules/hid_forward.c [hunk 3]
```diff
@@ -318,16 +404,177 @@ static u8_t hidc_read_cfg(struct bt_gatt_hids_c *hidc,
...
+		err = bt_gatt_hids_c_rep_read(recipient_hidc,
+					      config_rep,
+					      hidc_read_cfg);
+
+		if (err) {
+			LOG_ERR("Reading report failed, err:%d", err);
+			notify_config_forwarded(CONFIG_STATUS_WRITE_ERROR);
+		}
+
+		return;
```
Comment (pdunaj) [inline applications/nrf_desktop/src/modules/hid_forward.c]: return not needed

Comment (MarekPieta) [reply applications/nrf_desktop/src/modules/hid_forward.c]: done

File: applications/nrf_desktop/src/modules/hid_forward.c (hunk 2 above)
Comment (MarekPieta) [reply applications/nrf_desktop/src/modules/hid_forward.c]: done

File: applications/nrf_desktop/src/modules/hid_forward.c (hunk 1 above)
Comment (MarekPieta) [reply applications/nrf_desktop/src/modules/hid_forward.c]: checkpath got it already, fixed

**Pull request merged**